_includes/head.html

```diff
@@ -14,6 +14,17 @@
 
 	<meta name="title" content="{{ title }}">
 
+	<!-- include manifest-->
+	<link rel="manifest" href="/manifest.json">
```

Comment:
Can you remove the `/` before manifest.json here and see if this works fine ? Since the deploy url has `/2019/` at the end I was wondering if it will break there.